src/frontends/paddle/src/op_table.cpp

```diff
@@ -219,7 +219,6 @@ std::map<std::string, CreatorFunction> get_supported_ops() {
             {"relu", op::relu},
             {"relu6", op::relu6},
             {"reshape2", op::reshape2},
-            {"reverse", op::reverse},
```

Comment:
Why did you remove it? We need to preserve backward-compatibility for models programmed via previous version pdpd.

Reply:
> Why did you remove it? We need to preserve backward-compatibility for models programmed via previous version pdpd.

sorry, I have recovered this part of the code.